packages/typespec-ts/test/integration/generated/payload/pageable/.gitignore

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
It would be cool if we had a way to automate adding new cadl ranch specs so that .gitignore and config are automatically added :) just an idea for later